src/api.js

```diff
@@ -18,10 +18,10 @@ const fetchArticle = (article_id) => {
     });
 };
 
-const updateVotes = (article_id) => {
+const updateVotesArticle = (article_id, value) => {
   return axios
     .patch(`https://nc-be-news.herokuapp.com/api/articles/${article_id}`, {
-      inc_votes: 1,
+      inc_votes: value,
     })
     .catch((err) => {
       console.log(err);
```

Comment:
We're not doing anything with this error? We should be able to see your error component if the vote is unsuccessful for whatever reason.

Reply:
ooop sorry forgot the comment errors will do that now